base-rules/dell-redfish-v117.yaml

```diff
@@ -0,0 +1,104 @@
+Traverse:
+  Root: /redfish/v1
+  Excludes:
+    # Schemas
+    - /JsonSchemas
+    # Non-metric properties
+    - /Accounts
+    - /Certificates
+    - /Jobs
+    - /Logs
+    - /Registries
+    - /Roles
+    - /Sessions
+    - /Settings
+    # Services
+    - /AccountService
+    - /CertificateService
+    - /EventService
+    - /JobService
+    - /LogServices
+    - /SessionService
+    - /TaskService
+    - /TelemetryService
+    - /UpdateService
+    - /LicenseService  # Add v1.17
+    # Sub-metrics included in /Power or /Thermal
+    - /Power/
+    - /Power#/
+    - /Thermal#/
+    - /Assembly#/
+    # Redundant page with extra trailing slash
+    - /redfish/v1/Chassis/$
+
+Metrics:
+  ## Chassis
+  - Path: /redfish/v1/Chassis/{chassis}
+  - Path: /redfish/v1/Chassis/{chassis}/Assembly
+  - Path: /redfish/v1/Chassis/{chassis}/NetworkAdapters/{function}
+  - Path: /redfish/v1/Chassis/{chassis}/NetworkAdapters/{function}/NetworkDeviceFunctions/{nic}
+  - Path: /redfish/v1/Chassis/{chassis}/NetworkAdapters/{function}/NetworkPorts/{port}
+
+  - Path: /redfish/v1/Chassis/{chassis}/PCIeSlots
+  - Path: /redfish/v1/Chassis/{chassis}/Power
+  - Path: /redfish/v1/Chassis/{chassis}/Sensors/{sensor}
+  - Path: /redfish/v1/Chassis/{chassis}/Thermal
+  # Added for v1.17
+  - Path: /redfish/v1/Chassis/{chassis}/PCIeDevices/{device}
+  - Path: /redfish/v1/Chassis/{chassis}/PCIeDevices/{device}/PCIeDevices/{function}
```

Comment:
This path seems somewhat odd.
I guess this path could be a copy of `/redfish/v1/Chassis/{chassis}/PCIeDevices/{device}`.
If it is a meaningful path, we should choose a better label than `{function}`.

Reply:
I check the collector generated JSON files. but I could not find this kind of Path. I remove it.